include/swift/AST/ModuleDependencies.h

```diff
@@ -348,8 +348,14 @@ class SwiftBinaryModuleDependencyStorage : public ModuleDependencyInfoStorageBas
   /// The path to the .swiftSourceInfo file.
   const std::string sourceInfoPath;
 
-  /// The paths of all the .pch dependencies of this module.
-  const std::vector<std::string> preCompiledBridgingHeaderPaths;
+  /// The paths of all the .h dependencies of this module.
+  const std::vector<std::string> headerImports;
+
+  /// Source files on which the header inputs depend.
```

Comment:
What is `headerSourceFiles` vs. `headerImports` and when they will be different? I think we only need one that covers all the .h file that is not part of other modules?

Reply:
`headerSourceFiles` are all the files that get included from files in `headerImports`. I'm open to other names. 😬 

We already had `BridgingHeaderSourceFiles` for this, so I opted to match that here. Although not immediately useful, I figured scanner clients like CAS may in the future need this information?

Reply:
I am confused. You can only have one bridging header, aka one headerImports. And when you run dependency scanning for the translation unit on that one bridging header, you get all non modular includes, which includes the original bridging header. I feel that that is only set of headers you need to keep track of, no?